src/content/docs/more-integrations/dbt-cloud/dbt-cloud-integration.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Dbt Cloud with Airflow and Snowflake Integration 
+title: Dbt Cloud with Airflow and Snowflake integration 
```

Comment:
@paperclypse Should this be "dbt" (lower case)? The product name appears to be lower case in their [docs](https://www.getdbt.com/product/what-is-dbt) and we use "dbt" in the doc itself.